_posts/2022-08-03-installing-gdb.md

```diff
@@ -3,6 +3,7 @@ title: "Tools we use: installing GDB for ARM"
 description: Demonstrate a few methods of installing gdb for ARM
 author: noah
 image: img/installing-gdb/cover.png # 1200x630
+tags: [ARM, gdb]
```

Comment:
should tags be all lowercase?

Reply:
We had some previous tags that were uppercase, but I think I was just being lazy. I'll change them to all be lowercase.